docs/source/conf.py

```diff
@@ -81,9 +81,9 @@
 TAGS = []
 smv_tag_whitelist = multiversion_regex_builder(TAGS)
 # Whitelist pattern for branches (set to None to ignore all branches)
-BRANCHES = ['master', 'v1.12', 'v1.13', 'v1.14']
```

Comment:
should we drop 1.12?

Reply:
not now, I'll drop it once v1.15 is stable.

Reply:
ok